### ISSUE TITLE
Incorrect screen resolution

### ISSUE THREAD

**Issue by christopheg** [elgalu/docker-selenium#124]
#### Operating System
- [x ] I provided my operating system name and version, e.g. output of `uname -a`
  Linux 1258da611a0c 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2+deb8u3 (2016-07-02) x86_64 x86_64 x86_64 GNU/Linux
#### Image version
- [ x] I have latest version of this image. Upgrade with `docker pull elgalu/selenium`
#### Docker version
- [x ] If I don't have latest version of docker I will specify which version with `docker --version` output
  Docker version 1.12.0, build 8eab29e

I'm starting the docker image with these options:

``` sh
docker run -d --name=selenium -p 4444:24444 -p 5900:25900 \
  -e TZ="Europe/Brussels" -e VNC_PASSWORD=XXXX \
  -e MAX_INSTANCES=20 -e MAX_SESSIONS=20 \
  -e SCREEN_WIDTH=1080 -v /dev/shm:/dev/shm elgalu
```

However the screen resolution seems incorrect. In attachment you can see the result:

![elgalu](https://cloud.githubusercontent.com/assets/199087/19304855/a4f0db20-906e-11e6-89af-310ecf101f00.png)

**Comment by elgalu** [comment]
Hi @christopheg thanks for the feedback, seems I introduced a recent bug, in the meantime you can avoid this problem by passing `-e RC_CHROME=false -e RC_FIREFOX=false` to docker run:

```sh
docker run ... -e RC_CHROME=false -e RC_FIREFOX=false
```

To disable the legacy selenium IDE RC driver.

**Comment by ebaranov** [comment]
@elgalu I've added `RC_CHROME` parameter as you suggest, but got an error:

`Error: Format string '%(ENV_RC_CHROME)s' for 'program:selenium-rc-chrome.autostart' contains names ('ENV_RC_CHROME') which cannot be expanded. Available names: ENV_BIN_UTILS, ENV_BROWSERSTACK_STOP_SIGNAL, ENV_BSTACK_ACCESS_KEY, ENV_BSTACK_LOG_FILE, ENV_BSTACK_TUNNEL, ENV_BSTACK_TUNNEL_ID, ENV_BSTACK_TUNNEL_OPTS, ENV_BSTACK_TUNNEL_URL, ENV_BSTACK_TUNNEL_ZIP, ENV_BSTACK_WAIT_RETRY_TIMEOUT, ENV_BSTACK_WAIT_TIMEOUT, ENV_CHROME, ENV_CHROME_ARGS, ENV_CHROME_BASE_DEB_PATH, ENV_CHROME_DRIVER_BASE, ENV_CHROME_DRIVER_FILE, ENV_CHROME_FLAVOR, ENV_CHROME_PATH, ENV_CHROME_VERBOSELOGGING, ENV_CHROME_VERSION, ENV_COMMON_CAPS, ENV_CONTAINER_IP, ENV_CPU_ARCH, ENV_CUSTOM_SELENIUM_NODE_PROXY_PARAMS, ENV_CUSTOM_SELENIUM_NODE_REGISTER_CYCLE, ENV_DBUS_SESSION_BUS_ADDRESS, ENV_DBUS_SESSION_BUS_PID, ENV_DEBCONF_NONINTERACTIVE_SEEN, ENV_DEBIAN_FRONTEND, ENV_DEFAULT_NOVNC_PORT, ENV_DEFAULT_SAUCE_LOCAL_SEL_PORT, ENV_DEFAULT_SELENIUM_HUB_PORT, ENV_DEFAULT_SELENIUM_NODE_CH_PORT, ENV_DEFAULT_SELENIUM_NODE_FF_PORT, ENV_DEFAULT_SELENIUM_NODE_RC_CH_PORT, ENV_DEFAULT_SELENIUM_NODE_RC_FF_PORT, ENV_DEFAULT_SSHD_PORT, ENV_DEFAULT_SUPERVISOR_HTTP_PORT, ENV_DEFAULT_VNC_PORT, ENV_DISABLE_ROLLBACK, ENV_DISPLAY, ENV_DISP_N, ENV_DOCKER_HOST_IP, ENV_DOCKER_SELENIUM_STATUS, ENV_DOCKER_SOCK, ENV_ENCODING, ENV_ETHERNET_DEVICE_NAME, ENV_FFMPEG_CODEC_ARGS, ENV_FFMPEG_FRAME_RATE, ENV_FFMPEG_FRAME_SIZE, ENV_FF_BASE_URL, ENV_FF_COMP, ENV_FF_INNER_PATH, ENV_FF_LANG, ENV_FF_PLATFORM, ENV_FF_URL, ENV_FF_VER, ENV_FIREFOX, ENV_FIREFOX_DEST_BIN, ENV_FIREFOX_VERSION, ENV_GECKOD_URL, ENV_GECKOD_VER, ENV_GEOMETRY, ENV_GREP_ONLY_NUMS_VER, ENV_GRID, ENV_HOME, ENV_HOSTNAME, ENV_HUB_ENV_no_proxy, ENV_LANG, ENV_LANGUAGE, ENV_LANG_WHERE, ENV_LANG_WHICH, ENV_LOGFILE_BACKUPS, ENV_LOGFILE_MAXBYTES, ENV_LOGS_DIR, ENV_LOG_LEVEL, ENV_MAX_DISPLAY_SEARCH, ENV_MAX_INSTANCES, ENV_MAX_SESSIONS, ENV_MEM_JAVA_PERCENT, ENV_NORMAL_GROUP, ENV_NORMAL_USER, ENV_NORMAL_USER_GID, ENV_NORMAL_USER_HOME, ENV_NORMAL_USER_UID, ENV_NOVNC, ENV_NOVNC_PORT, ENV_NOVNC_STOP_SIGNAL, ENV_NSS_USE_SHARED_DB, ENV_PATH, ENV_PICK_ALL_RANDMON_PORTS, ENV_PWD, ENV_RUN_DIR, ENV_SAUCELABS_STOP_SIGNAL, ENV_SAUCE_API_KEY, ENV_SAUCE_CONN_DOWN_URL, ENV_SAUCE_CONN_VER, ENV_SAUCE_LOCAL_SEL_PORT, ENV_SAUCE_LOG_FILE, ENV_SAUCE_TUNNEL, ENV_SAUCE_TUNNEL_DOCTOR_TEST, ENV_SAUCE_TUNNEL_ID, ENV_SAUCE_TUNNEL_MAX_RETRY_ATTEMPTS, ENV_SAUCE_TUNNEL_READY_FILE, ENV_SAUCE_USER_NAME, ENV_SAUCE_WAIT_RETRY_TIMEOUT, ENV_SAUCE_WAIT_TIMEOUT, ENV_SCREEN_DEPTH, ENV_SCREEN_HEIGHT, ENV_SCREEN_MAIN_DEPTH, ENV_SCREEN_NUM, ENV_SCREEN_SUB_DEPTH, ENV_SCREEN_WIDTH, ENV_SELENIUM_HUB_HOST, ENV_SELENIUM_HUB_PARAMS, ENV_SELENIUM_HUB_PORT, ENV_SELENIUM_HUB_PROTO, ENV_SELENIUM_HUB_STOP_SIGNAL, ENV_SELENIUM_JAR_PATH, ENV_SELENIUM_NODE_CHROME_STOP_SIGNAL, ENV_SELENIUM_NODE_CH_PORT, ENV_SELENIUM_NODE_FF_PORT, ENV_SELENIUM_NODE_FIREFOX_STOP_SIGNAL, ENV_SELENIUM_NODE_HOST, ENV_SELENIUM_NODE_PARAMS, ENV_SELENIUM_NODE_PROXY_PARAMS, ENV_SELENIUM_NODE_RC_CH_PORT, ENV_SELENIUM_NODE_RC_FF_PORT, ENV_SELENIUM_NODE_REGISTER_CYCLE, ENV_SEL_BROWSER_TIMEOUT_SECS, ENV_SEL_CLEANUPCYCLE_MS, ENV_SEL_DIRECTORY, ENV_SEL_HOME, ENV_SEL_MAJOR_MINOR_VER, ENV_SEL_NODEPOLLING_MS, ENV_SEL_PATCH_LEVEL_VER, ENV_SEL_RELEASE_TIMEOUT_SECS, ENV_SEL_UNREGISTER_IF_STILL_DOWN_AFTER, ENV_SEL_VERSION, ENV_SHLVL, ENV_SHM_SIZE, ENV_SHM_TRY_MOUNT_UNMOUNT, ENV_SLEEP_SECS_AFTER_KILLING_SUPERVISORD, ENV_SSHD, ENV_SSHD_GATEWAYPORTS, ENV_SSHD_PORT, ENV_SSHD_STOP_SIGNAL, ENV_SSHD_X11FORWARDING, ENV_SUPERVISOR_HTTP_PASSWORD, ENV_SUPERVISOR_HTTP_PORT, ENV_SUPERVISOR_HTTP_USERNAME, ENV_SUPERVISOR_KILLASGROUP, ENV_SUPERVISOR_NOT_REQUIRED_SRV_LIST1, ENV_SUPERVISOR_NOT_REQUIRED_SRV_LIST2, ENV_SUPERVISOR_PIDFILE, ENV_SUPERVISOR_REQUIRED_SRV_LIST, ENV_SUPERVISOR_STOPASGROUP, ENV_SUPERVISOR_STOPSIGNAL, ENV_SUPERVISOR_STOPWAITSECS, ENV_TAIL_LOG_LINES, ENV_TEST_SLEEPS, ENV_TZ, ENV_UBUNTU_DATE, ENV_UBUNTU_FLAVOR, ENV_USER, ENV_USE_SELENIUM, ENV_VIDEO, ENV_VIDEOS_DIR, ENV_VIDEO_CHUNKS_MAX, ENV_VIDEO_CHUNK_SECS, ENV_VIDEO_FILE_EXTENSION, ENV_VIDEO_FILE_NAME, ENV_VIDEO_FLUSH_SECS, ENV_VIDEO_LOG_FILE, ENV_VIDEO_PATH, ENV_VIDEO_PIDFILE, ENV_VIDEO_REC_STOP_SIGNAL, ENV_VNC2SWF_ENCODING, ENV_VNC2SWF_FRAMERATE, ENV_VNC_CLI_OPTS, ENV_VNC_FROM_PORT, ENV_VNC_PASSWORD, ENV_VNC_PORT, ENV_VNC_START, ENV_VNC_STOP_SIGNAL, ENV_VNC_STORE_PWD_FILE, ENV_VNC_TO_PORT, ENV_VNC_TRYOUT_ERR_LOG, ENV_VNC_TRYOUT_OUT_LOG, ENV_WAIT_FOREGROUND_RETRY, ENV_WAIT_TIMEOUT, ENV_WAIT_VNC_FOREGROUND_RETRY, ENV_XEPHYR_DISPLAY, ENV_XMANAGER, ENV_XMANAGER_STARTRETRIES, ENV_XMANAGER_STARTSECS, ENV_XMANAGER_STOP_SIGNAL, ENV_XTERM_STOP_SIGNAL, ENV_XVFB_CLI_OPTS_BASE, ENV_XVFB_CLI_OPTS_EXT, ENV_XVFB_CLI_OPTS_TCP, ENV_XVFB_STARTRETRIES, ENV_XVFB_STOP_SIGNAL, ENV__, ENV_no_proxy, group_name, here, host_node_name, program_name in section 'program:selenium-rc-chrome' (file: '/etc/supervisor/conf.d/selenium-rc-chrome.conf')`

**Comment by ebaranov** [comment]
@elgalu everything works well, thank you! I got an error above, since I used incorrect format for describing ENV params in docker-compose.yaml. Everything working well for me, with the configuration below, hope it help someone too:

```
environment:
  - TZ=US/Pacific
  - SCREEN_WIDTH=1200
  - SCREEN_HEIGHT=800
  - RC_CHROME=false
  - RC_FIREFOX=false
```

**Comment by elgalu** [comment]
Thanks for reporting back @ebaranov , I've just released [3.0.0c](https://github.com/elgalu/docker-selenium/releases/tag/3.0.0c) which should fix this issue and #125 among other things.

@christopheg if you have a moment please `docker pull elgalu/selenium` to get latest an re-test.

**Comment by ebaranov** [comment]
@elgalu checked out today, works well without `-RC_CHROME=false`, ` - RC_FIREFOX=false` only with:

```
environment:
  - TZ=US/Pacific
  - SCREEN_WIDTH=1200
  - SCREEN_HEIGHT=800
```
